.github/workflows/next-lint-check.yml

```
@@ -0,0 +1,31 @@
+name: Next.js Linting Check
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  lint:
+    name: Linting with Next.js
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

```suggestion
          node-version: '20'
```
It should support the [latest LTS nodejs version](https://github.com/nodejs/release?tab=readme-ov-file#release-schedule)